src/DRICH_geo.cpp

```diff
@@ -384,17 +388,31 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
     // cut overlaps with other sectors using "pie slice" wedges, to the extent specified
     // by `mirrorPhiw`
     Tube              pieSlice(0.01 * cm, vesselRmax2, tankLength / 2.0, -mirrorPhiw / 2.0, mirrorPhiw / 2.0);
+    Tube              pieSlice1(0.01 * cm, vesselRmax2, tankLength / 2.0, -mirrorPhiw / 2.0, (0.05*mirrorPhiw) / 2.0);
+    Tube              pieSlice2(0.01 * cm, vesselRmax2, tankLength / 2.0, (0.06*mirrorPhiw) / 2.0, mirrorPhiw / 2.0);
     IntersectionSolid mirrorSolid2(pieSlice, mirrorSolid1, mirrorPlacement);
+    IntersectionSolid mirrorSolid4(pieSlice1, mirrorSolid3, mirrorPlacement);
+    IntersectionSolid mirrorSolid5(pieSlice2, mirrorSolid3, mirrorPlacement);
 
     // mirror volume, attributes, and placement
-    Volume mirrorVol(detName + "_mirror_" + secName, mirrorSolid2, mirrorMat);
+    Volume mirrorVol(detName + "_mirror_0" + secName, mirrorSolid2, mirrorMat);
+    Volume mirrorVol2(detName + "_mirror_1" + secName, mirrorSolid4, mirrorMat);
+    Volume mirrorVol3(detName + "_mirror_2" + secName, mirrorSolid5, mirrorMat);
```

Comment:
```suggestion
    Volume mirrorVol(detName + "_mirror_tile0" + secName, mirrorSolid2, mirrorMat);
    Volume mirrorVol2(detName + "_mirror_tile1" + secName, mirrorSolid4, mirrorMat);
    Volume mirrorVol3(detName + "_mirror_tile2" + secName, mirrorSolid5, mirrorMat);
```
or some suitable name, just so it's clear what `0`, `1`, and `2` mean.